lib/smart_proxy_openscap/fetch_scap_file.rb

```diff
@@ -24,22 +24,18 @@ def download_path
         "api/v2/compliance/policies/:policy_id/content"
       when :tailoring_file
         "api/v2/compliance/policies/:policy_id/tailoring"
-      when :oval_content
-        "api/v2/compliance/oval_policies/:policy_id/oval_content"
       end
     end
 
     def file_name(policy_id, digest)
       case @type
       when :scap_content, :tailoring_file
         "#{policy_id}_#{digest}.xml"
-      when :oval_content
-        "#{digest}.oval.xml.bz2"
```

Comment:
We could also remove this from the test data: https://github.com/theforeman/smart_proxy_openscap/blob/master/test/data/oval-results.xml.bz2

Reply:
Dropped this and one more